eslint.shared.config.mjs

```diff
@@ -138,7 +138,7 @@ export default [
       '@o3r/json-dependency-versions-harmonize': [
         'error',
         {
-          ignoredPackages: ['@o3r/build-helpers'],
+          ignoredPackages: ['@o3r/build-helpers', '@o3r/workspace-helpers'],
```

Comment:
Not needed here, `@o3r/workspace-helpers` should be used only at workspace level